cmd/icinga-notifications-daemon/main.go

```diff
@@ -60,6 +61,9 @@ func main() {
 		utils.PrintErrorThenExit(err, 1)
 	}
 
+	// When Icinga Notifications is started by systemd, we've to notify systemd that we're ready.
+	_ = sdnotify.Ready()
+
 	logger := logs.GetLogger()
 	defer func() { _ = logger.Sync() }()
 
```

Comment:
I think this is put quite early and provides no real benefit over not sending the ready notification and just using `Type=simple`. Typically, with systemd, services should be report readiness when they can accept requests. So for our case, this would probably be after the listening socket has been opened.